packages/admin/src/pages/dashboard/index.page.tsx

```diff
@@ -20,13 +21,18 @@ export const getServerSideProps = async ({
   query,
 }: GetServerSidePropsContext) => {
   const paginationParams: Pagination = {
-    paginate: true,
+    paginate: false,
     page: 0,
     size: 2,
     sort: 'createdDate,DESC',
   };
   const sessionCookie = getSessionIdFromCookies(req);
-  const schemes = await getUserSchemes(paginationParams, sessionCookie);
+  const allUserSchemes = await getOwnedAndEditableSchemes(
+    paginationParams,
+    sessionCookie
+  );
+  const schemesUserOwns = allUserSchemes.ownedSchemes;
+  const schemesUserCanEdit = allUserSchemes.editableSchemes;
```

Comment:
minor point but could be one line here

```suggestion
const { ownedSchemes: schemesUserOwns, editableSchemes: schemesUserCanEdit } =
  await getOwnedAndEditableSchemes(paginationParams, sessionCookie);
```
personally i'd keep the names returned by the service (`ownedSchemes` & `editableSchemes`) as component props, but have renamed them above anyways

Reply:
Will make this change now, much cleaner!